packages/core-contracts/contracts/ownership/OwnableStorage.sol

```diff
@@ -5,6 +5,7 @@ contract OwnableStorage {
     struct OwnableStore {
         address owner;
         address nominatedOwner;
+        bool initialized;
```

Comment:
Should we put all these at the beginning of namespaces before we go live into any real networks?

Reply:
done